### PR TITLE
SEO-173273 - WinUI Weblink Controls

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @MallikaRK,
Worked on image alt text and renamed images.

Regards,
Sharly A.

### CODE REVIEW

File: winui/Scheduler/Getting-Started.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ this.Schedule.ViewType = SchedulerViewType.Month;
 {% endhighlight %}
 {% endtabs %}
 
-![show-month-view-in-winui-scheduler](GettingStarted_Images/month-view-in-winui-scheduler.png)
+![Show month view in WinUI Scheduler](gettingstarted_images/winui-scheduler-show-month-view.png)
```
Comment (MallikaRK) [inline winui/Scheduler/Getting-Started.md:81]: Alt text is Fine. 
But existing image name is fine. revert your changes in image name alone for all scheduler change.

File: winui/Scheduler/Getting-Started.md [hunk 2]
```diff
@@ -399,4 +399,4 @@ Schedule.FirstDayOfWeek = DayOfWeek.Monday;
 {% endhighlight %}  
 {% endtabs %}  
 
-![change-first-day-of-week-in-winui-scheduler](GettingStarted_Images/change-first-day-of-week-in-winui-scheduler.png)
\ No newline at end of file
+![Change first day of week in WinUI Scheduler](gettingstarted_images/winui-scheduler-change-first-day-of-week.png)
```
Comment (MallikaRK) [inline winui/Scheduler/Getting-Started.md:402]: here too.

File: winui/Time-Picker/Getting-Started.md [hunk 3]
```diff
@@ -108,7 +108,7 @@ sfTimePicker.SelectedTime = new DateTimeOffset(new DateTime(2021, 10, 29, 10, 45
 {% endhighlight %}
 {% endtabs %}
 
-![programmatic-time-selection-in-winui-time-picker](Getting-Started_images/programmatic-time-selection-in-winui-time-picker.png)
```
Comment (MallikaRK) [inline winui/Time-Picker/Getting-Started.md:111]: Existing image names are fine. So, revert your image name changes in all places.

File: winui/Time-Picker/Time-Restriction.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ sfTimePicker.MaxTime = new DateTimeOffset(new DateTime(DateTime.Now.Year, DateTi
 {% endhighlight  %}
 {% endtabs %}
 
-![change-minimum-and-maximum-times-in-winui-time-picker](Getting-Started_images/change-minimum-and-maximum-times-in-winui-time-picker.png)
+![Change minimum and maximum times in WinUI TimePicker](getting-started_images/winui-time-picker-change-minimum-and-maximum-times.png)
```
Comment (MallikaRK) [inline winui/Time-Picker/Time-Restriction.md:28]: This files too, revert your image name changes.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: winui/Scheduler/Getting-Started.md (hunk 2 above)
Comment (Sharly-Atieno) [reply winui/Scheduler/Getting-Started.md]: Changed

File: winui/Time-Picker/Getting-Started.md (hunk 3 above)
Comment (Sharly-Atieno) [reply winui/Time-Picker/Getting-Started.md:111]: changed

File: winui/Time-Picker/Time-Restriction.md (hunk 4 above)
Comment (Sharly-Atieno) [reply winui/Time-Picker/Time-Restriction.md]: Changed

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**